src/main/resources/reference.conf

```diff
@@ -1290,20 +1290,45 @@ resourceTypes = {
       "unlock_resource" = {
         description = "Can unlock a resource"
       }
+      "read_aggregate_data" = {
+        description = "Can read aggregate data about a snapshot"
+      }
+      "create_snapshot_request" = {
+        description = "Can create a snapshot request for this snapshot"
+      }
+      "get_snapshot_builder_settings" = {
+        description = "Can get the snapshot builder settings for this snapshot"
+      }
+      "update_snapshot_builder_settings" = {
+        description = "Can get the snapshot builder settings for this snapshot"
+      }
     }
     ownerRoleName = "steward"
     roles = {
       steward = {
-        roleActions = ["delete", "edit_datasnapshot", "update_snapshot", "read_data",  "discover_data",  "share_policy::steward", "share_policy::custodian", "share_policy::reader",  "share_policy::discoverer", "read_policies", "set_public", "update_passport_identifier", "export_snapshot", "view_journal", "read_auth_domain", "update_auth_domain", "lock_resource", "unlock_resource"]
+        roleActions = ["share_policy::steward", "share_policy::custodian", "update_passport_identifier", "view_journal"]
+        includedRoles = ["custodian"]
       }
       custodian = {
-        roleActions = ["delete", "edit_datasnapshot", "update_snapshot", "read_data",  "discover_data",  "share_policy::reader",  "share_policy::discoverer", "read_policies", "set_public", "export_snapshot", "read_auth_domain", "update_auth_domain", "lock_resource", "unlock_resource"]
+        roleActions = ["delete", "edit_datasnapshot", "update_snapshot", "share_policy::reader",  "share_policy::discoverer", "read_policies", "set_public", "update_auth_domain", "lock_resource", "unlock_resource"]
+        includedRoles = ["reader"]
+      }
+      snapshot_builder_manager = {
+        roleActions = ["update_snapshot_builder_settings"]
+        descendantRoles = {
+          snapshot_builder_request = ["approver"]
+        }
+      }
+      aggregate_data_reader = {
+        roleActions = ["read_aggregate_data", "create_snapshot_request"]
+        includedRoles = ["discoverer"]
       }
       discoverer = {
-        roleActions = ["discover_data", "read_policy::steward", "read_policy::discoverer", "read_auth_domain"]
+        roleActions = ["discover_data", "read_policy::steward", "read_policy::discoverer", "read_auth_domain", "get_snapshot_builder_settings"]
       }
       reader = {
-        roleActions = ["read_data", "discover_data", "read_policy::steward", "read_policy::custodian", "read_policy::discoverer", "export_snapshot", "read_auth_domain"]
```

Comment:
(For my own understanding)

The following roles are now included via `discoverer` (`aggregate_data_reader` includes `discoverer`):
```
discover_data
read_policy::steward
read_policy::discoverer
read_auth_domain
```